examples/pxScene2d/src/pxTextBox.cpp

```diff
@@ -388,25 +388,14 @@ void pxTextBox::measureTextWithWrapOrNewLine(const char *text, float sx, float s
     int i = 0;
     int lasti = 0;
     int numbytes = 1;
-    char* tempChar = NULL;
+    //char* tempChar = NULL;
```

Comment:
Please remove this code instead of commenting it out (since it is no longer needed).

Reply:
Thanks Mike. I have cleaned this code.